docs/_articles/en/einstein/einstein-troubleshoot.md

```diff
@@ -26,12 +26,15 @@ Use this troubleshooting checklist to identify and eliminate common problems wit
 2. Check that your Einstein for Developers extension is activated using the **Developer: Show Running Extensions** command. If your extension isn’t activating, file an [issue](https://github.com/forcedotcom/Einstein-GPT-for-Developers/issues) in our GitHub repo.
 3. Make sure you’re connected to a Salesforce org. The status of your default org connection is visible in the status bar.
 
-
 ![Default Org](./images/default-org.png)
 
-
 **Tip**: Run the **SFDX: Open Default Org** command to easily check that your Salesforce extensions are running correctly and that you're connected to a Salesforce org.
 
+### Code Builder Environment Checklist
 
+Einstein for Developers is disabled by default in new Code Builder environments. Telemetry is used to audit enablement of Salesforce Generative AI functionality and must be on when you enable the extension:
 
+1. Go to **Settings** and search for `salesforcedx-vscode-core.telemetry.enabled` to confirm that this Salesforce telemetry setting in on.
+2. Click the Einstein icon in the status bar and enable the extension.
```

Comment:
Same update here, re step 2/3. It would also be ok to just put that step here if you think that's better!